common/app/views/fragments/atoms/youtube.scala.html

```diff
@@ -24,20 +25,27 @@
 
 @defining(media.activeAssets.headOption) { activeAsset: Option[MediaAsset] =>
 @defining(media.expired.getOrElse(false)){expired: Boolean =>
-@defining(playable && !posterImageOverride.exists(_ => YouTubePosterOverride.isSwitchedOn)){sixteenByNine: Boolean =>
-
-
-    <div data-media-atom-id="@media.id" class="@RenderClasses(Map(
-        ("u-responsive-ratio", true),
-        ("u-responsive-ratio--hd", sixteenByNine),
-        ("youtube-media-atom", true),
-        ("no-player", !playable || expired )
-    ))"
-    @for(endSlatePath <- media.endSlatePath if displayEndSlate)  {
-      data-end-slate="@endSlatePath"
+    @defining(playable && !posterImageOverride.exists(_ => YouTubePosterOverride.isSwitchedOn)){sixteenByNine: Boolean =>
+        @defining(
+            if (YouTubeRelatedVideos.isSwitchedOn) {
+                media.channelId.getOrElse("UCHpw8xwDNhU9gdohEcJu4aA")
```

Comment:
This is the ID of the main [Guardian YouTube channel](https://www.youtube.com/channel/UCHpw8xwDNhU9gdohEcJu4aA). But maybe we don't need a default?